src/App.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable*/
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
```

Comment:
- [ ] Please, consider implementing the router, as requered: 

- Has two `Route` s, one for the `BookList` component and the other for the `Categories` component
In order to not show a blank page when visiting the site for the first time, the BookList component should be the default page

```
    <Route exact path="/">
      <BookList />
    </Route>
```

You have to implement the navigation: 

- Should have 2 `<Link>`s in the navbar which leads to the booklist and categories locations